ggshield/cmd/main.py

```diff
@@ -136,12 +159,6 @@ def cli(
         # we still have to setup logs
```

Comment:
This part on the other hand seems already handled in the callback ?

Reply:
You mean `debug_callback()`? It only handles the `--debug` argument, not `debug: true` being set in the configuration file.

Reply:
My comment was not clear. I mean that before the changes, we used to set `config.debug` to the value passed as an option if it was not None. This is not the case in the new version, we only setup_debug_logs but don't set the debug value in the `config`. Is this normal?  
In the end, this comment is the same as [this one](https://github.com/GitGuardian/ggshield/pull/411/commits/bc5fc94000000908ef6717a5dcdf966efe788088#r1016629305).